src/oaklib/converters/owl_to_fhir.py

```diff
@@ -0,0 +1,25 @@
+"""Convert OWL to FHIR CodeSystem JSON
+
+TODO's
+  1. Write functioning converter
+  2. Optional/Later: Schema mappings (Obograph -> FHIR CodeSystem). SchemaAutomator? Other approach?
+  3. Add converter to CLI
+  4. Test(s)
+
+todo's (minor)
+  1. owl_to_fhir.py -> owl_to_fhir_codesystem.py?
+"""
+from typing import Dict
+
+
+def convert(inpath: str, outpath: str) -> Dict:
+    """Convert
+
+    Side effects:
+    - Writes file at `outpath`"""
```

Comment:
is the content the same as Dict? I would choose one or the other

[sorry to comment on a draft I can't help it!]

Reply:
Rather than writing to dicts you could have an actual python datamodel for the FHIR code system and serialize this. This has numerous advantages such as type safety, clarity, ease of refactoring. See the src/datamodels folder for examples

Reply:
Commenting on the draft is good. The input is useful and it could take a long time to complete this PR if I get pulled in other directions.

This is a sound suggestion. I've added it to the list of "todo's minor" at the top of the file. My primary objective is to complete this issue for TIMS so that I can continue working on other issues, but if I can improve the typing at all in a quick way, I will do that earlier on.